dist/css.js

```diff
@@ -6,6 +6,22 @@ const PREFERS_COLOR_SCHEME_REGEXP =
 
 const bodies = $WPT_BODIES;
 
+function countExternalCssInHead() {
+  return document.querySelectorAll( 'head link[rel="stylesheet"]' ).length;
+}
+
+function countInlineCssInHead() {
+  return document.querySelectorAll( 'head style' ).length;
+}
+
+function countExternalCssAfterHead() {
+  return document.querySelectorAll( 'link[rel="stylesheet"]' ).length - countExternalCssInHead();
+}
+
+function countInlineCssAfterHead() {
+  return document.querySelectorAll( 'style' ).length - countInlineCssInHead();
```

Comment:
Ditto

```suggestion
function countInlineCssInBody() {
  return document.querySelectorAll( 'body style' ).length;
```

Reply:
See above, fixed in https://github.com/HTTPArchive/custom-metrics/pull/64/commits/70a1cda990ffc87c50bca29342656b9690dda48e